kolibri/plugins/device_management/assets/src/views/available-channels-page/index.vue

```diff
@@ -6,34 +6,36 @@
       :errorType="wizardStatus"
     />
 
-    <section
-      v-if="channelsAreAvailable"
-      class="top-matter"
-    >
-      <div class="channels">
-        <h1 class="channels-header">
-          <span v-if="inExportMode">{{ $tr('yourChannels') }}</span>
-          <span v-else-if="inLocalImportMode">{{ selectedDrive.name }}</span>
-          <span v-else>{{ $tr('channels') }}</span>
-        </h1>
-
-        <p>{{ $tr('channelsAvailable', { channels: availableChannels.length }) }}</p>
-      </div>
+    <h1>
+      <span v-if="inExportMode">{{ $tr('yourChannels') }}</span>
+      <span v-else-if="inLocalImportMode">{{ selectedDrive.name }}</span>
+      <span v-else>{{ $tr('channels') }}</span>
+    </h1>
 
-      <div class="filters">
+    <k-grid v-if="channelsAreAvailable" class="top-matter">
+      <k-grid-item sizes="4, 8, 4">
+        <p :class="{ 'text-offset': windowIsLarge }">
+          {{ $tr('channelsAvailable', { channels: availableChannels.length }) }}
+        </p>
+      </k-grid-item>
+      <k-grid-item sizes="4, 3, 3" alignments="left, left, right">
```

Comment:
It's common to need to change the alignment of grid item contents at different breakpoints. This shorthand notation allows that: `alignments="left, left, right"`.

Here, content is aligned left on small and medium windows, and right on large windows.